samples/Java/azure-servicebus/QueuesWithProxy/src/main/java/com/microsoft/azure/servicebus/samples/queueswithproxy/QueuesWithProxy.java

```diff
@@ -0,0 +1,247 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.samples.queueswithproxy;
+
+import com.google.gson.reflect.TypeToken;
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.google.gson.Gson;
+
+import static java.nio.charset.StandardCharsets.*;
+
+import java.io.IOException;
+import java.net.*;
+import java.time.Duration;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.function.Function;
+
+import com.microsoft.azure.servicebus.primitives.StringUtil;
+import org.apache.commons.cli.*;
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class QueuesWithProxy {
+
+    static final Gson GSON = new Gson();
+
+    public void run(String connectionString) throws Exception {
+        // Set the transport type to AmqpWithWebsockets
+        ConnectionStringBuilder connStrBuilder = new ConnectionStringBuilder(connectionString, "BasicQueue");
+        /* TODO requires proxy update to build */
+        // connStrBuilder.setTransportType(TransportType.AMQP_WEB_SOCKETS);
```

Comment:
Please remove the commented part referring to AMQP Web sockets unless needed.

Reply:
It's definitely needed, I had meant to uncomment it. This sample is using a preview JAR now so it can actually use proxy, previously it was waiting for a JAR that included proxy support. I will open another PR for that line.